templates/fargate.cfn.yml

```diff
@@ -291,7 +299,7 @@ Resources:
               pre_build:
                   commands:
                   - printenv
-                  - TAG="$ENVIRONMENT_NAME.$(date +%Y-%m-%d).$(echo $CODEBUILD_RESOLVED_SOURCE_VERSION | head -c 8)"
+                  - TAG="$REPOSITORY_NAME.$REPOSITORY_BRANCH.$ENVIRONMENT_NAME.$(date +%Y-%m-%d.%H.%M.%S).$(echo $CODEBUILD_RESOLVED_SOURCE_VERSION | head -c 8)"
```

Comment:
I'd also say, if you're going to concatenate like this, you may want to use variable expansion to make it easier for somebody to skim, and break out some of the variables into their own variables for easier readability